src/app/video-games/detail/components/button-toggle/button-toggle.component.scss

```diff
@@ -0,0 +1,31 @@
+button {
+  padding: 0.6rem 1.25rem;
+  border-radius: 0.35rem;
+  border: none;
+  font-size: 1.2em;
+  margin-left: 0.4rem;
+  margin-right: 0.4rem;
+  color: #FFF;
+  background: #F44336;
+  box-shadow: 0px 5px 0px 0px #D32F2F;
+
+  &:active {
+    box-shadow: 0px 2px 0px 0px #D32F2F;
+    transform: translateY(3px);
+  }
+
+  &.active {
```

Comment:
The Idea is that the colors are changed when the button is toggled, like so:

**Not toggled:**
![image](https://cloud.githubusercontent.com/assets/18172605/23314447/180af6dc-fa90-11e6-8448-e28cae7e0cd0.png)

**Toggled:**
![image](https://cloud.githubusercontent.com/assets/18172605/23314470/3373d6a0-fa90-11e6-940a-c3d0825c465c.png)

Perhaps creating `.toggle-off` and `.toggle-on` classes would be better?